com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -107,7 +107,12 @@ internal void __endSendServerRpc(NetworkSerializer serializer, uint rpcMethodId,
 #if DEVELOPMENT_BUILD || UNITY_EDITOR
```

Comment:
Not related to this PR, but does this mean we wont have access to stats in the RNSM in release builds outside of the editor?

Reply:
Only RPCs. There was some concern that the memory needed to keep RPC names around shouldn't exist, so we compromised with this strategy for now.